src/ralph/lib/table/table.py

```diff
@@ -0,0 +1,110 @@
+try:
+    from dj.choices import Choices
+    use_choices = True
+except ImportError:
+    Choices = None
+    use_choices = False
+
+from django.core.exceptions import FieldDoesNotExist
+from django.forms.utils import flatatt
+
+from ralph.admin.helpers import get_field_by_relation_path
+
+
+class TableRenderer(object):
+
+    """
+    Generate HTML tables for the defined configuration.
+
+    Example:
+        >>> table = TableRender(queryset, ['id', 'name'])
+        >>> table.render()
+
+
+
+
+        [['id', 'name'], ['1', 'test']]
+    """
+
+    def __init__(self, queryset, list_display, additional_row_method=None):
+        """
+        Initialize table class
+
+        :param queryset: Django queryset
+        :param list_display: Field list to display
+        :param additional_row_method: list of additional method for each row
+        """
+        self.queryset = queryset
+        self.list_display = list_display
+        self.additional_row_method = additional_row_method
```

Comment:
what if `additional_row_method` is not passed (you are iterating over it in `get_table_content`)? Shouldn't there be `or []`?